google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -2808,15 +2819,15 @@ public void testUpdateNonExistingTable() {
         TableInfo.of(
             TableId.of(DATASET, "test_update_non_existing_table"),
             StandardTableDefinition.of(SIMPLE_SCHEMA));
-    try {
-      bigquery.update(tableInfo);
-      fail("BigQueryException was expected");
-    } catch (BigQueryException e) {
-      BigQueryError error = e.getError();
-      assertNotNull(error);
-      assertEquals("notFound", error.getReason());
-      assertNotNull(error.getMessage());
-    }
+    BigQueryException exception =
+        assertThrows(
+            "BigQueryException was expected",
+            BigQueryException.class,
+            () -> bigquery.update(tableInfo));
+    BigQueryError error = exception.getError();
+    assertNotNull(error);
+    assertEquals("notFound", error.getReason());
+    assertNotNull(error.getMessage());
```

Comment:
Perhaps we could also simplify the remaining assert statements further by using the Truth.assertThat?
Example: assertThat(exception).hasMessageThat().contains("...");